packages/toolpad-app/src/toolpad/AppEditor/utils/normalizePageRowColumnSizes.ts

```diff
@@ -0,0 +1,54 @@
+import { NodeId } from '@mui/toolpad-core';
+import * as appDom from '../../../appDom';
+import { isPageRow } from '../../../runtime/toolpadComponents';
+
+const rowColumnCounts: Record<NodeId, number> = {};
+
+export default function normalizePageRowColumnSizes(
+  draftDom: appDom.AppDom,
```

Comment:
Perhaps we can remove the word `draft` from this function as the usage is more general now, i guess it could be just `dom` instead of `draftDom` for example?